apps/expo/src/examples/real-example.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { Alert, Text, TouchableOpacity, View } from 'react-native';
+import type { OptionType } from '@mobile-reality/react-native-select-pro';
+import { Select } from '@mobile-reality/react-native-select-pro';
+
+import { SafeAreaViewWrapper } from '../components/safe-area-view-wrapper';
+
+export const PROGRAMMING_LANGUAGES = [
+    {
+        value: 'javascript',
+        label: 'Javascript',
+    },
+    {
+        value: 'typescript',
+        label: 'Typescript',
+    },
+    {
+        value: 'java',
+        label: 'Java',
+    },
+    {
+        value: 'python',
+        label: 'Python',
+    },
+    {
+        value: 'c#',
+        label: 'C#',
+    },
+    {
+        value: 'c',
+        label: 'C',
+    },
+    {
+        value: 'c++',
+        label: 'C++',
+    },
+];
+
+export const RealExample = () => {
+    const { control, handleSubmit, formState, watch, reset } = useForm<{ languages: OptionType[] }>(
+        {
+            defaultValues: {
+                languages: [],
+            },
+        },
+    );
+
+    const { languages } = watch();
+
+    const onSubmit = () => {
+        Alert.alert('Your choices has been saved!');
+    };
+
+    return (
+        <SafeAreaViewWrapper
+            style={{
+                backgroundColor: '#000a19',
+                padding: 25,
+                flex: 1,
+                paddingTop: 200,
+                justifyContent: 'space-between',
+            }}
+        >
+            {formState.isSubmitted ? (
+                <View style={{ width: '100%' }}>
+                    <Text
+                        style={{
+                            fontSize: 16,
+                            color: '#f34c54',
+                            marginBottom: 10,
+                        }}
+                    >
+                        Your programming languages are:
+                    </Text>
+                    {languages.length > 0 &&
+                        languages.map((language, index) => (
+                            <Text
+                                key={index}
+                                style={{ fontSize: 16, color: '#04e590', marginBottom: 10 }}
+                            >
+                                {language.label}
+                            </Text>
+                        ))}
+                </View>
+            ) : (
+                <View style={{ width: '100%' }}>
+                    <Text style={{ fontSize: 16, color: '#f34c54', marginBottom: 10 }}>
+                        Select your programming languages
+                    </Text>
+                    <Controller
```

Comment:
please use `useController` instead `Controller`. `useController` is more readable and based on hooks instead render props